crates/types/src/motion_selection.rs

```diff
@@ -48,6 +50,8 @@ impl Default for MotionType {
 
 #[derive(Clone, Debug, Deserialize, Serialize, PathSerialize, PathDeserialize, PathIntrospect)]
 pub struct MotionSafeExits {
+    animation: bool,
+    animationstiff: bool,
```

Comment:
is it possible to have only one field `animation` in here?
If not, please rename `animationstiff` to `animation_stiff`